course_inventory/inventory.py

```diff
@@ -99,7 +100,7 @@ def gather_course_data_from_api(account_id: int, term_id: int) -> pd.DataFrame:
     logger.info(f'Course Page Number: {page_num}')
     response = make_request_using_api_utils(url_ending_with_scope, params)
     all_course_data = json.loads(response.text)
-    slim_course_dicts = slim_down_course_data(all_course_data)
+    course_dicts = slim_down_course_data(all_course_data)
```

Comment:
Here and in the following lines I just shortened some variable names, since some of them were ridiculously long.